### PR TITLE
jupyterlab 3.5.1

### PR DESCRIPTION
**Pull request by miccal**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
ResourceMissingError: jupyterlab does not define resource "jupyter-console"
```

**Comment by cho-m** [comment]
May want to check on significant increase in build time. ARM build used to take 30-40min but is now going for 90min+.

**Comment by chenrui333** [comment]
I wonder if it is `jupyter-console` thing (the previous build on 13-arm was in the 30-40min range)

File: Formula/jupyterlab.rb [hunk 1]
```diff
@@ -198,13 +203,13 @@ class Jupyterlab < Formula
   end
 
   resource "notebook" do
-    url "https://files.pythonhosted.org/packages/d5/94/b15c0e44c37e49cf77866ff56cc7644632229b79c113a0eafd908fc7c7d7/notebook-6.4.12.tar.gz"
-    sha256 "6268c9ec9048cff7a45405c990c29ac9ca40b0bc3ec29263d218c5e01f2b4e86"
+    url "https://files.pythonhosted.org/packages/41/15/a4285abb25c87dec3002cd7eab88320ef21448effdd3714840695aafab12/notebook-6.5.2.tar.gz"
+    sha256 "c1897e5317e225fc78b45549a6ab4b668e4c996fd03a04e938fe5e7af2bfffd0"
```
Comment (cho-m) [inline Formula/jupyterlab.rb:207]: Build may be having trouble due to new `notebook` having build dependency on `nbclassic` which has a dependency on `jupyter-core`/`jupyter-server`.

**Pull request closed without merging**